### ISSUE TITLE
File Conversion Issues using LibreOffice 4.1

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2529]
Originally reported on Google Code with ID 1778

```
Reported by Andy Landsbury in bigbluebutton-dev

  https://groups.google.com/d/msg/bigbluebutton-dev/K30rj0YvDCg/h6XQLPxQrV4J

Andy provides sample file to reproduce the problem with LibreOffice 4.1 converting
this document.  

Some pages of the document have the converted text going white.  In other pages the
size of the fonts are too large in BigBlueButton and not all text is visible.


```

Reported by `ffdixon` on 2014-08-07 16:50:05

<hr>
- _Attachment: [Example File.pptx](https://storage.googleapis.com/google-code-attachments/bigbluebutton/issue-1778/comment-0/Example File.pptx)_

**Comment by GhaziTriki** [comment]
All slides from the powerpoint files are now converted in a correct format, expect the slide 10 where coloured boxes sizes are not matching. A comparison image could be found below

![image](https://cloud.githubusercontent.com/assets/4991088/13707514/5050c188-e7aa-11e5-9f7c-b59bdf8ac277.png)

**Comment by ffdixon** [comment]
Marking as fixed -- tested with BigBlueButton 1.1-beta (which uses libreoffice 5.1.4) converts the slides correctly.
